### DISCUSSION TITLE
r/optimization-upgrade-for-the-v80-rv

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#11] [Announcements]
# r/optimization-upgrade-for-the-v80-rv

房车使用一年我对房车的改装

https://blog.ops-coffee.cn/r/optimization-upgrade-for-the-v80-rv

**Comment by iyuanze** [comment]
这是什么牌子的房车啊，哪个型号，图片看不很清楚

**Reply by d081** [reply]
大通V80

**Comment by Poole33** [comment]
有没有改装工作室推荐吖，或者搜索渠道也行，坐标上海，感谢🙏

**Reply by ops-coffee** [reply]
没有，改装这个要看运气，毕竟非标品，工作室或者改装厂首先考虑的是赚钱，效果因师父收益跟心情决定，推荐自己动手，如果自己不行的话，考虑成品
